package.json

```diff
@@ -75,12 +75,14 @@
   "devDependencies": {
     "coveralls": "^2.11.8",
     "eslint": "^3.4.0",
+    "fs-extra": "^0.30.0",
```

Comment:
This is a super older version but it's the last version that supports Node 0.10.